src/Command/SyncThumbsCommand.php

```diff
@@ -161,8 +161,9 @@ public function execute(InputInterface $input, OutputInterface $output): int
             }
 
             //clear entity manager for saving memory
-            \assert($this->mediaManager instanceof BaseManager);
-            $this->mediaManager->getObjectManager()->clear();
+            if ($this->mediaManager instanceof ClearableManagerInterface) {
```

Comment:
I assume you plan to make the PR on doctrine-extensions. Ping me when done ;)

Reply:
Yes!

Reply:
PR done, needs to be merged and released. Resolving this conversation.